packages/fxa-shared/test/sentry/pii-filters.ts

```diff
@@ -0,0 +1,256 @@
+      CommonPiiActions.depthFilter,
+      CommonPiiActions.urlUsernamePassword,
+      CommonPiiActions.emailValues,
+      CommonPiiActions.piiKeys,
+      CommonPiiActions.tokenValues,
+      CommonPiiActions.ipV4Values,
+      CommonPiiActions.ipV6Values,
+      new PiiRegexFilter(/foo/gi),
+    ]);
+
+    it('filters empty event', () => {
+      let event: Sentry.Event = {};
+      event = sentryFilter.filter(event);
+      expect(event).to.deep.equal({});
+    });
+
+    it('filters event', () => {
+      let event: Sentry.Event = {
+        message: 'A foo message.',
+        breadcrumbs: [
+          {
+            message: 'A foo breadcrumb',
+            data: {
+              first_name: 'foo',
+              last_name: 'bar',
+            },
+          },
+          {
+            message: 'A fine message',
+          },
+        ],
+        request: {
+          url: 'http://me:123@foo.bar/?email=foxkey@mozilla.com&uid=12345678123456781234567812345678',
+          query_string: {
+            email: 'foo',
+            uid: 'bar',
+          },
+          cookies: {
+            user: 'foo:bar',
+          },
+          env: {
+            key: '--foo',
+          },
+          headers: {
+            foo: 'a foo header',
+            bar: 'a foo bar bar header',
+            'oidc-claim': 'claim1',
+          },
+          data: {
+            info: {
+              email: 'foxkeh@mozilla.com',
+              uid: '12345678123456781234567812345678',
+            },
+            time: new Date(0).getTime(),
+          },
+        },
+        exception: {
+          values: [
+            {
+              value:
+                'Foo bar! A user with email foxkeh@mozilla.clom and ip 127.0.0.1 encountered an err.',
+            },
+          ],
+        },
+        extra: {
+          meta: {
+            email: 'foo@bar.com',
+          },
+          l1: {
+            l2: {
+              l3: {
+                l4: {
+                  l5: {
+                    l6: {
+                      l7: {
+                        l8: {
+                          l9: {
+                            l10: 'bar',
+                          },
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+        user: {
+          meta: {
+            email: 'foo@bar.com',
+          },
+          id: 'foo123',
+          ip_address: '127.0.0.1',
+          email: 'foo@bar.com',
+          username: 'foo.bar',
+        },
+        type: 'transaction',
+        spans: undefined, // Not testing, let's be careful not put PII in spans,
+        measurements: undefined, // NA, just numbers
+        debug_meta: undefined, // NA, image data
+        sdkProcessingMetadata: undefined, // NA, not used
+      };
+
+      event = sentryFilter.filter(event);
+
+      expect(event).to.deep.equal({
+        message: 'A [Filtered] message.',
```

Comment:
Q: Not sure if all these `[Filtered]` should use the imported `${FILTERED}` consts.